assets/sass/components/global/_googlesitekit-selection-box.scss

```diff
@@ -0,0 +1,76 @@
+/**
+ * SelectionBox styles.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-selection-box {
+	border: 2px solid $c-neutral-n-66;
+	border-radius: 8px;
+	outline: 1px solid $c-white;
+	outline-offset: -1px;
```

Comment:
Using a slightly out-of-the-box trick here with `outline` to "_mask_" one pixel of the border width.

Since the focus state border width is 2px, the border width from 1px to 2px causes a slight jump. This trick prevents this problem.

Reply:
For visibility, I'm adding screencasts.

Without this solution:

https://github.com/google/site-kit-wp/assets/20284937/06bd2896-38d8-418a-a50b-6abff248a96a

With this solution:

https://github.com/google/site-kit-wp/assets/20284937/ae7b1b65-00c5-40cf-ba5d-eabc3fafe8fd

Reply:
Sounds good, thanks for the detailed explanation!

Reply:
Incidentally, I think you probably mean _outside-the-box_ (or maybe _out-of-the-ordinary_) rather than _out-of-the-box_ in your description above, with "out of the box" generally meaning a ready-made, standard sort of thing, whereas "outside the box" is a reference to "thinking outside the box" i.e. taking a novel approach to something, which seems more like what's going on here :)

Reply:
> "outside the box" is a reference to "thinking outside the box" i.e. taking a novel approach to something, which seems more like what's going on here :)

Yes, that's what I meant. Thank you! 😄